### ISSUE TITLE
Container Annotation Fragments do not update accordingly while typing

### ISSUE THREAD

**Issue by michael** [substance/archivist-writer#10]
Probably related to our new containerannotation index.

**Comment by obuchtala** [comment]
The solution is to not use static Fragment objects, but ones that delegate dynammically to anno.startOffset etc.

**Comment by obuchtala** [comment]
Working on it.

**Comment by obuchtala** [comment]
Fixed with substance/substance@33ba6a8c4e5687939d3f16633b8aba77db6c5ded

**Comment by michael** [comment]
The error is gone.. however, there still seems to be a problem with getContainerAnnotationsForSelection. When i move the cursor over the remark in the 2nd paragraph it should return the remark annotation but the array is empty.

``` js
var annos = doc.getContainerAnnotationsForSelection(sel, contentContainer, {
  type: "remark"
});
console.log('remark annos for selection', annos);
```

Pushed debug output.

**Comment by obuchtala** [comment]
Will be fixed via #17
